src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -250,6 +250,12 @@ internal static partial class CSharpCodeStyleOptions
             "csharp_style_prefer_local_over_anonymous_function",
             $"TextEditor.CSharp.Specific.{nameof(PreferLocalOverAnonymousFunction)}");
 
+        public static readonly Option2<CodeStyleOption2<bool>> PreferTupleSwap = CreateOption(
+            CSharpCodeStyleOptionGroups.ExpressionLevelPreferences, nameof(PreferTupleSwap),
+            defaultValue: s_trueWithSuggestionEnforcement,
+            "csharp_style_prefer_tuple_swap",
+            $"TextEditor.CSharp.Specific.{nameof(PreferTupleSwap)}");
+
```

Comment:
In case this option will remain, this needs to be added to automation object to be persisted correctly.

Reply:
done.